### PR TITLE
Replace Microsoft Windows Update tokens

### PR DESCRIPTION
**Pull request by nxtn**

### CODE REVIEW

File: docs/framework/wpf/wpf-security-strategy-platform-security.md [hunk 1]
```diff
@@ -47,7 +47,7 @@ The core of Windows provides several security features that form the security fo
  [!INCLUDE[TLA2#tla_wpf](../../../includes/tla2sharptla-wpf-md.md)] is compiled with the /GS flag to add yet another layer of defense to [!INCLUDE[TLA2#tla_wpf](../../../includes/tla2sharptla-wpf-md.md)] applications.  
   
 #### Microsoft Windows Update Enhancements  
- [!INCLUDE[TLA#tla_win_update](../../../includes/tlasharptla-win-update-md.md)] was also improved in [!INCLUDE[TLA2#tla_winxpsp2](../../../includes/tla2sharptla-winxpsp2-md.md)] to simplify the process for downloading and installing updates. These changes significantly enhance security for [!INCLUDE[TLA2#tla_wpf](../../../includes/tla2sharptla-wpf-md.md)] customers by helping to ensure that their systems are up-to-date, particularly with respect to security updates.  
+ Microsoft Windows Update was also improved in [!INCLUDE[TLA2#tla_winxpsp2](../../../includes/tla2sharptla-winxpsp2-md.md)] to simplify the process for downloading and installing updates. These changes significantly enhance security for [!INCLUDE[TLA2#tla_wpf](../../../includes/tla2sharptla-wpf-md.md)] customers by helping to ensure that their systems are up-to-date, particularly with respect to security updates.
```
Comment (Youssef1313) [inline docs/framework/wpf/wpf-security-strategy-platform-security.md]: Should this whole header be removed based on #5199 ?

File: docs/framework/wpf/wpf-security-strategy-platform-security.md [hunk 2]
```diff
@@ -47,7 +47,7 @@ The core of Windows provides several security features that form the security fo
  [!INCLUDE[TLA2#tla_wpf](../../../includes/tla2sharptla-wpf-md.md)] is compiled with the /GS flag to add yet another layer of defense to [!INCLUDE[TLA2#tla_wpf](../../../includes/tla2sharptla-wpf-md.md)] applications.  
   
 #### Microsoft Windows Update Enhancements  
- [!INCLUDE[TLA#tla_win_update](../../../includes/tlasharptla-win-update-md.md)] was also improved in [!INCLUDE[TLA2#tla_winxpsp2](../../../includes/tla2sharptla-winxpsp2-md.md)] to simplify the process for downloading and installing updates. These changes significantly enhance security for [!INCLUDE[TLA2#tla_wpf](../../../includes/tla2sharptla-wpf-md.md)] customers by helping to ensure that their systems are up-to-date, particularly with respect to security updates.  
+ Microsoft Windows Update was also improved in [!INCLUDE[TLA2#tla_winxpsp2](../../../includes/tla2sharptla-winxpsp2-md.md)] to simplify the process for downloading and installing updates. These changes significantly enhance security for [!INCLUDE[TLA2#tla_wpf](../../../includes/tla2sharptla-wpf-md.md)] customers by helping to ensure that their systems are up-to-date, particularly with respect to security updates.  
   
 <a name="Windows_Vista"></a>
```
Comment (Youssef1313) [inline docs/framework/wpf/wpf-security-strategy-platform-security.md]: It seems that this anchor isn't used anywhere.

```suggestion
```

**Review by Youssef1313 — CHANGES_REQUESTED** [review]

File: docs/framework/wpf/wpf-security-strategy-platform-security.md (hunk 1 above)
Comment (NextTurn) [reply docs/framework/wpf/wpf-security-strategy-platform-security.md]: Thanks for pointing this out.

File: docs/framework/wpf/wpf-security-strategy-platform-security.md (hunk 2 above)
Comment (NextTurn) [reply docs/framework/wpf/wpf-security-strategy-platform-security.md]: It seems that none of the anchors in this article is referenced anywhere.

**Review by mairaw — APPROVED** [review]
Thanks @NextTurn. My suggestion: let's keep these PRs focused on the replacement of tokens. Getting them to be up to date will be way more involved and these odd anchors that we got from the MSDN migration can be a separate task. That way, the PRs are easy for us to review and merge.

**Pull request merged**